### PR TITLE
Fix VTX ready status

### PR DESCRIPTION
**Pull request by haslinghuis**
Status was no longer retrieved after saving

### CODE REVIEW

**Comment by HThuren** [comment]
Great, thank you :-)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Comment by nerdCopter** [comment]
when `False`, still green

File: src/js/tabs/vtx.js
Comment (haslinghuis) [inline src/js/tabs/vtx.js:42]:
```suggestion
        const isReady = !vtx.isVtxDeviceStatusNotReady();
```

**Comment by nerdCopter** [comment]
briefly still green while temporarily `false`, when saving already good config
https://youtu.be/p7bcBMJc8B0
`a229baa9`

**Comment by haslinghuis** [comment]
Last commit removes initial status, so on save it it does not flip.

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
